Phonebook.Frontend/proxy.conf.json

```
@@ -1,4 +1,16 @@
 {
+  "/signin-wsfed":{
+    "target": "https://localhost:5001",
+    "secure": false,
+    "changeOrigin": true,
+    "logLevel": "debug"
+  },
+  "/Login": {
```

can you make this route lowercase? (everywhere)
```suggestion
  "/login": {
```

---

Maybe I find a way to do it case insensitive. This was only a quick fix. But I agree the route is wrong. But in the whole file 🤷‍♀️ urls should be case insensitive